database/factories/ChargesFactory.php

```diff
@@ -0,0 +1,26 @@
+<?php
+
+namespace Database\Factories;
+
+use App\Models\Bill;
+use Illuminate\Database\Eloquent\Factories\Factory;
+
+/**
+ * @extends \Illuminate\Database\Eloquent\Factories\Factory<\App\Models\Charge>
+ */
+class ChargesFactory extends Factory
+{
+    /**
+     * Define the model's default state.
+     *
+     * @return array<string, mixed>
+     */
+    public function definition(): array
+    {
+        return [
+            'bill_id' => Bill::factory(),
+            'chargeable_id' => $this->faker->numberBetween(1, 999), // see the comment in PR
+            'chargeable_type' => $this->faker->word,
```

Comment:
Please refer to polymorphic relationship to see how this works

Reply:
Besides this generic structure with a random chargeable assigned, you could also have different structures for each type of chargeable specifically.